test/acvp_test.c

```diff
@@ -1479,6 +1484,7 @@ int setup_tests(void)
 #endif /* OPENSSL_NO_DH */
 
 #ifndef OPENSSL_NO_DSA
+    dsasign_allowed = fips_provider_version_lt(libctx, 3, 4, 0);
```

Comment:
This should be 3.1.0 shouldn't it?

Reply:
Don't think it matters.

Reply:
I thought we target 3.4 for these changes..